libgeopmd/include/geopm_public.h

```diff
@@ -14,3 +14,17 @@
  */
 #define GEOPM_PRIVATE __attribute__((visibility("hidden")))
 #endif
+
+/*
+ * String length allocated for GEOPM names (like signal and control
+ * names). Has same value as NAME_MAX from limits.h for historical
+ * reasons.
+ */
+#define GEOPM_NAME_MAX 255
+
+/*
+ * String length allocated for GEOPM messages (like C error messages
+ * derived from C++ exceptions).  Has same value as PATH_MAX from
+ * linux/limits.h for historical reasons.
+ */
+#define GEOPM_MESSAGE_MAX 4096
```

Comment:
Since this is used for geopm error message strings should it go in geopm_error.h instead?

Reply:
Both of these macros are also used for other string lengths than the primary use case.  We could add a geopm_limits.h.  I believe that geopm_public.h is the only public header that defines preprocessor macros.  Perhaps this is reason to leave them in geopm_public.h?